docs/operations/karpenter.md

```diff
@@ -45,16 +45,16 @@ A Karpenter-managed InstanceGroup controls a corresponding Karpenter Provisioner
 
 Note that not all features of InstanceGroups are supported.
 
+## Subnets
+
+By default, kOps will tag subnets with `kops.k8s.io/instance-group/<intancegroup>: "true"` for each InstanceGroup the subnet is assigned to. If you enable manual tagging of subnets, you have to ensure you add these tags are added, if not Karpenter will fail to provision any instances.
```

Comment:
```suggestion
By default, kOps will tag subnets with `kops.k8s.io/instance-group/<intancegroup>: "true"` for each InstanceGroup the subnet is assigned to. If you enable manual tagging of subnets, you have to ensure these tags are added, if not Karpenter will fail to provision any instances.
```


maybe add `{{ kops_feature_table(kops_added_default='1.25') }}` too, or whatever version this is intended to land in

Reply:
It's feature flagged so thinking this is not necessary.